src/main/java/org/broadinstitute/consent/http/service/ResearcherService.java

```diff
@@ -95,7 +91,8 @@ private User validateAuthUser(AuthUser authUser) {
 
     private List<UserProperty> describeResearcherProperties(Integer userId) {
         validateUser(userId);
-        return userPropertyDAO.findResearcherPropertiesByUser(userId);
+        return userPropertyDAO.findResearcherPropertiesByUser(userId,
+                Arrays.stream(UserFields.values()).map(UserFields::getValue).collect(Collectors.toList()));
```

Comment:
Something I like to do is add utility methods to enums to help clean up inline arguments like this. For example, if `UserFields` had this method:
```
  public static List<String> getValues() {
    return Stream.of(UserFields.values()).map(UserFields::getValue).collect(Collectors.toList());
  }
```
then this dao call could be streamlined:
```
return userPropertyDAO.findResearcherPropertiesByUser(userId, UserFields.getValues());
```